docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -68,6 +68,7 @@ This file contains several service definitions:
   task instances once their dependencies are complete.
 - ``airflow-webserver`` - The webserver is available at ``http://localhost:8080``.
 - ``airflow-worker`` - The worker that executes the tasks given by the scheduler.
+- ``airflow-triggerer`` - The triggers runs an asyncio event loop.
```

Comment:
```suggestion
- ``airflow-triggerer`` - The triggers runs an async event loop.
```

Reply:
Are we sure, we have to go with the mention of `deferrable tasks`?

Reply:
The triggerer is designed specifically for deferrable tasks, and IIRC this term is already used elsewhere in documentation (or is it deferrrable operators? Task and operator are interchangable in most cases thouse)